modules/luci-mod-network/htdocs/luci-static/resources/view/network/dhcp.js

```diff
@@ -340,6 +343,66 @@ return view.extend({
 		o.optional = true;
 		o.placeholder = 'loopback';
 
+		o = s.taboption('relay', form.SectionValue, '__relays__', form.TableSection, 'relay', null,
+			_('Relay DHCP requests elsewhere. OK: v4<->v4, v6<->v6. Not OK: v4<->v6, v6<->v4.')
+			+ '<br />' + _('Note: you may also need a DHCP Proxy (currently unavailable) when specifying a non-standard Relay To port(<code>addr#port</code>).')
+			+ '<br />' + _('You may add multiple unique Relay To on the same Listen addr.'));
+
+		ss = o.subsection;
+
+		ss.addremove = true;
+		ss.anonymous = true;
+		ss.sortable  = true;
+		ss.rowcolors = true;
+		ss.nodescriptions = true;
+
+		so = ss.option(form.Value, 'id', _('ID'));
+		so.rmempty = false;
+		so.optional = true;
+
+		so = ss.option(widgets.NetworkSelect, 'interface', _('Interface'));
+		so.optional = true;
+		so.rmempty = false;
+		so.placeholder = 'lan';
+
+		so = ss.option(form.Value, 'local_addr', _('Listen addr'));
+		so.rmempty = false;
+		so.datatype = 'ipaddr';
+
+		for (var family = 4; family <= 6; family += 2) {
+			for (var i = 0; i < ndevs.length; i++) {
+				var addrs = (family == 6) ? ndevs[i].getIP6Addrs() : ndevs[i].getIPAddrs();
+				for (var j = 0; j < addrs.length; j++)
+					so.value(addrs[j].split('/')[0]);
+			}
+		}
+
+		so = ss.option(form.Value, 'server_addr', _('Relay To'));
```

Comment:
Could we not change the translation for the uci option `local_addr` to `Listen address` and for the uci option `server_addr` to `Relay address`? 

That is more common.
See also the other translations in this pullrequest where `RelayTo` and `Listen addr` are used.
https://github.com/openwrt/luci/pull/6047/files#diff-90caf036d2316d858be5df6b382c44d4130a66c52cc7b6fda8ee715971a4f9bdR368
https://github.com/openwrt/luci/pull/6047/files#diff-90caf036d2316d858be5df6b382c44d4130a66c52cc7b6fda8ee715971a4f9bdR403
https://github.com/openwrt/luci/pull/6047/files#diff-90caf036d2316d858be5df6b382c44d4130a66c52cc7b6fda8ee715971a4f9bdR397

Reply:
Relay address seems more ambiguous in this context as a relay. "Is this the from or to?" "Which side is it?"
I can certainly update to "Listen address", however.